src/main/drivers/light_led.c

```diff
@@ -44,11 +44,18 @@ static uint8_t ledInversion = 0;
 #define LED2_PIN NONE
 #endif
 
+#if defined(USE_RX_SPI) && !defined(RX_SPI_LED_PIN)
```

Comment:
Not sure if dragging `USE_RX_SPI` in here is a good idea - we could make this into LED3, and allow other drivers to use it as well - if the driver using the functionality uses an index for the LED to use, conflicts can be dealt with in the config.

Reply:
OK, I need to think of a way to implement that, I like the idea of an additional led that can by used by other drivers as well. But ultimately for a given target, that led should serve just one purpose IMO. I'm not sure how to ensure this.
I would still prefer to declare it in config.h for f3 targets as SPI_RX_LED_PIN and SPI_RX_LED_INVERTED instead of LED_3*.

Reply:
Ok, I have given this some more thought as well. The correct way to use parameter groups is to make them as much self contained as possible. In this case that means to have the config for the spi rx led in the spi rx parameter group. This will make it harder to integrate the code handling it, but it will elegantly solve the problem of only making the led available if spi rx is supported.